resources/js/components/Kiosk.vue

```diff
@@ -268,25 +287,8 @@ export default {
                     }
                 })
                 .catch(error => {
-                    if (error.response.status === 403) {
-                        this.$swal.fire({
-                            title: 'Whoops!',
-                            text: "You don't have permission to perform that action.",
-                            icon: 'error',
-                        });
-                    } else if (error.response.status === 401) {
-                        this.$swal.fire({
-                            title: 'Whoops!',
-                            text: "Invalid API token or authentication error",
-                            icon: 'error',
-                        });
-                    } else {
-                        this.$swal.fire({
-                            title: 'Error',
-                            text: 'Unable to process data. Check your internet connection or try refreshing the page.',
-                            icon: 'error',
-                        });
-                    }
+                    this.axiosErrorToBugsnag(error)
+                    this.axiosErrorToBugsnag(error)
```

Comment:
Is this intentionally duplicated?

Reply:
I think this should be `handleAxiosError`.

Reply:
No